README.md

```diff
@@ -13,14 +13,15 @@ sets trasmitted invoice to paid in Alma
 
 ## Required ENV
 
-- `SENTRY_DSN` = If set to a valid Sentry DSN, enables Sentry exception monitoring. This is not needed for local development.
-- `WORKSPACE` = Set to `dev` for local development, this will be set to `stage` and `prod` in those environments by Terraform.
+
 - `ALMA_API_URL` = Base URL for making Alma API calls
 - `ALMA_API_READ_WRITE_KEY` = API key for making Alma API calls
+- `LOG_LEVEL`
 - `SAP_DROPBOX_CONNECTION` = JSON formatted connection information for accessing SAP dropbox
 - `SAP_REPLY_TO_EMAIL`
```

Comment:
I would go ahead and write descriptions for each of these.

Reply:
what is SES_SEND_FROM_EMAIL?

Reply:
The email address that SES sends from, e.g. `noreply@libraries.mit.edu`